modules/150-user-authn/docs/FAQ.md

```diff
@@ -2,15 +2,15 @@
 title: "The user-authn module: FAQ"
 ---
 
+{% raw %}
+
 ## How to secure my application?
 
 It is possible to hide your application behind Dex authentication by using the `DexAuthenticator` custom resource (CR).
 In fact, by creating the DexAuthenticator in a cluster, user creates an instance [oauth2-proxy](https://github.com/oauth2-proxy/oauth2-proxy), which is already connected to Dex.
```

Comment:
To enable Dex authentication for an application, follow these steps:
1. Create a custom resource [DexAuthenticator](cr.html#dexauthenticator) ?